packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts

```diff
@@ -56,7 +56,11 @@ context('Access to pagelist', () => {
       cy.get('button.close').eq(1).click();
     });
 
-    cy.screenshot(`${ssPrefix}8-close-page-list-modal`, {capture: 'viewport'});
+    // Hide release & license on home page , page-list count and page creation & date
+    cy.screenshot(`${ssPrefix}8-close-page-list-modal`, {
+      capture: 'viewport',
+      blackout: ['[data-line="2"]:eq(0) > a > img', '[data-hide-in-vrt=true]', '[data-testid="pageListButton"] > span']
+    });
```

Comment:
Why is this diff here?
It looks like this is not related to `page list timeline`